examples/src/legacy/multiple_kms_cmk_regions.py

```diff
@@ -0,0 +1,64 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+Example showing basic encryption and decryption of a value already in memory
+using multiple KMS CMKs in multiple regions.
+"""
+import aws_encryption_sdk
+from aws_encryption_sdk.key_providers.kms import KMSMasterKey, KMSMasterKeyProvider
+
+
+def run(aws_kms_generator_cmk, aws_kms_child_cmks, source_plaintext, botocore_session=None):
+    """Encrypts and then decrypts a string under multiple KMS customer master keys (CMKs) in multiple regions.
+
+    :param str aws_kms_generator_cmk: Amazon Resource Name (ARN) of the primary KMS CMK
+    :param List[str] aws_kms_child_cmks: Additional Amazon Resource Names (ARNs) of secondary KMS CMKs
+    :param bytes source_plaintext: Data to encrypt
+    :param botocore_session: existing botocore session instance
+    :type botocore_session: botocore.session.Session
+    """
+    child_cmk = aws_kms_child_cmks[0]
+    # Check that these keys are in different regions
+    assert not aws_kms_generator_cmk.split(":")[3] == child_cmk.split(":")[3]
+
+    kwargs = dict(key_ids=[aws_kms_generator_cmk, child_cmk])
+
+    if botocore_session is not None:
+        kwargs["botocore_session"] = botocore_session
+
+    # Create master key provider using the ARNs of the keys and the session (botocore_session)
+    kms_key_provider = KMSMasterKeyProvider(**kwargs)
```

Comment:
If botocore_session is None, is the `KMSMasterKeyProvider` step still safe to perform? E.g. Is a new session created? Does this fail and crash? etc.

Reply:
In my next-step cleanup, I'll be removing that as an option in the example (and probably adding an example of how to provide a custom session instead). If you don't provide a session, the KMS MKP will automatically create one for you.